client/components/tinymce/plugins/simple-payments/dialog/product-image-picker.jsx

```diff
@@ -114,7 +110,6 @@ class ProductImagePicker extends Component {
 						enabledFilters={ [ 'images' ] }
 						visible={ isSelecting }
 						isBackdropVisible={ false }
-						disabledDataSources={ [ 'pexels', 'google_photos' ] }
```

Comment:
Maybe worth cleaning up `disabledDataSources` from `MediaModal` and `MediaLibrary` as well, since it's not used anymore?
(In a follow up I mean)

Reply:
I was thinking about that but decided to leave it there as it might be useful in the future. If we decide that we want to remove it, I'd be happy to clean it up.